x-pack/plugins/lens/public/indexpattern_datasource/indexpattern_suggestions.ts

```diff
@@ -665,7 +677,7 @@ function createSuggestionWithDefaultDateHistogram(
     label: i18n.translate('xpack.lens.indexpattern.suggestions.overTimeLabel', {
       defaultMessage: 'Over time',
     }),
-    changeType: 'extended',
+    changeType: shouldSwitchTimeWithBuckets ? 'reorder' : 'extended',
```

Comment:
This suggestion should _always_ be extended because it should always add a new column. Calling it a reorder will affect how the XY charts use the suggestion, which I don't think is intentional.

Reply:
You're right. All the code in this function needs to be removed as it was shaped to make it work the initial way.